source/support/check-organisation-email-address.html.erb

```diff
@@ -0,0 +1,56 @@
+---
+title: Check your organisation email address - GovWifi
+
+tags:
+
+description: Check your government or public sector email address to sign up to GovWifi.
+---
+
+<div class="container">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <%= partial 'shared/support/sidebar' %>
+    </div>
+    <div class="column-two-thirds">
+      <h1>Check your organisation email address</h1>
+      <p>You can <%= link_to "signup to GovWifi", "https://www.wifi.service.gov.uk/support/connect-to-govwifi", class: "govuk-link" %>
+        using your government or public sector email address
+      </p>
+      <p>If you don't have a government or public sector email address, you can
+        <%= link_to "connect to GovWifi as a visitor", "https://www.wifi.service.gov.uk/support/connect-visitors-to-govwifi", class: "govuk-link" %>
+      </p>
+      <p>You can check to see if your organisation email is eligible below:</p>
+      <div class="govuk-form-group">
+        <input class="govuk-input input-field-margin-bottom" id="search-table" type="text" placeholder="Search email domains">
+      </div>
+
+      <% email_domains = ["police.uk", "nhs.uk", "cjsm.net", "nhs.net", "ejudiciary.net", "hmcts.net", "wmfs.net", "highwaysengland.co.uk", "deltservices.co.uk", "ukfi.co.uk","uksbs.co.uk",
```

Comment:
Can we refactor this out into a config or yaml or ruby file? Also we should be programatically sorting it alphabetically with something like `email_domains.sort`

Reply:
See https://middlemanapp.com/advanced/data-files/